.pipelines/azure-pipeline-build.yml

```diff
@@ -820,6 +820,47 @@ jobs:
         pathToPublish: '$(Build.ArtifactStagingDirectory)'
         artifactName: drop
 
+- job: Helm_Chart
```

Comment:
Is this for the deployment to the Dev Cluster?

Reply:
yes, since we removed package helm chart, this was failing. added this back